juju/juju
# Goyaml - move to a fork instead of patch file

## Description of change

The following commit removes the patch file for go-yaml race
issues and instead depends on a fork with the fixes applied. This
way we can make sure that the source of juju is correct, because
the source code of go-yaml is applied at source.

## QA steps

 1. Ensure that all tests pass
 1. Run race detector to see that races have disappeared for yaml.

## Documentation changes

Developers of juju should no longer provide patch files for upstream
changes and instead use `dep` source re-writing to do the same job.

## Bug reference

https://bugs.launchpad.net/juju/+bug/1795007
